views/default/forms/login.php

```diff
@@ -8,16 +8,18 @@
 ?>
 
 <div>
-	<label><?php echo elgg_echo('loginusername'); ?></label>
-	<?php echo elgg_view('input/text', array(
-		'name' => 'username',
-		'autofocus' => true,
-		));
-	?>
+	<label><?php echo elgg_echo('loginusername'); ?>
```

Comment:
Not sure we should touch the login form. Every project I have worked relies on a certain markup and styles this form heavily.